index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+<body>
+
+<section class="header">
+    
+    <div class="Top-Nav">
+        <div class="logo">
+          My Logo
+        </div>
+        <div class="Item-Nav">
+            
+            <a href="">Portfolio</a>
+            <a href="">About</a>
+            <a href="">Contact</a>
+        </div>
```

Comment:
- [ ] Since you are using a mobile-first approach to build this page. You should have a hamburger icon. 😄